core/src/main/scala/com/spotify/featran/FeatureBuilder.scala

```diff
@@ -31,6 +31,7 @@ object FeatureRejection {
   case class OutOfBound(lower: Double, upper: Double, actual: Double) extends FeatureRejection
   case class Unseen(labels: Set[String]) extends FeatureRejection
   case class WrongDimension(expected: Int, actual: Int) extends FeatureRejection
+  case class Outlier(actual: Double) extends FeatureRejection
```

Comment:
could also add lower/upper bound here, was not sure to add it since I am not sure it will generalize to all kind of Outlier rejectors